AirCasting/CoreData/Session.swift

```diff
@@ -143,6 +143,9 @@ public enum DeviceType: Int, CustomStringConvertible {
         case .AIRBEAM: return "AirBeam"
         }
     }
+    func isAirBeam() -> Bool {
```

Comment:
That's funny bc I literally searched for this in my code just before creating a PR because I thought I spotted it. And it wasn't there, and now I know why :extra-grimace: 
<img width="311" alt="Screenshot 2024-04-15 at 13 25 37" src="https://github.com/HabitatMap/AirCastingiOS/assets/46443841/9d49038b-a727-4733-87dc-15bb3a2f581a">